src/main/scala/interfaces/Afe.scala

```diff
@@ -56,17 +62,21 @@ class SidebandAfeIo(
     afeParams: AfeParams,
 ) extends Bundle {
 
+  val fifoParams = Input(new FifoParams())
+
   /** Data to transmit on the sideband.
     *
     * Output from the async FIFO.
     */
-  val txData = Decoupled(Bits(afeParams.sbSerializerRatio.W))
+  val txData = Input(UInt(afeParams.sbWidth.W))
```

Comment:
Why is tx an input? Same with rx being an output? I know you might have flipped it but makes readability hard.